lms/templates/fields/field_social_link_account.underscore

```diff
@@ -0,0 +1,13 @@
+<div class="u-field-value field <%- linkClass %>">
+    <label class="u-field-title field-label" for="u-field-link-<%- id %>-default"><%- title %></label>
```

Comment:
I am not seeing the user name under the social media title (like it is in the attached design) 
![screen shot 2016-05-10 at 1 46 54 pm](https://cloud.githubusercontent.com/assets/2052185/15156606/d1cffa46-16b5-11e6-8a0d-d3e821582cb7.png)